barretenberg/cpp/src/barretenberg/api/file_io.hpp

```diff
@@ -51,6 +52,7 @@ inline std::vector<uint8_t> read_file(const std::string& filename, size_t bytes
 
 inline void write_file(const std::string& filename, std::vector<uint8_t> const& data)
 {
+    info("writing file to ", filename);
```

Comment:
Do we want this to stay in? maybe verbose only?

Reply:
no removed it